packages/server/src/integrations/tests/sql.spec.ts

```diff
@@ -709,99 +267,4 @@ describe("SQL query builder", () => {
       sql: `insert into "test" ("name") values ($1) returning *`,
     })
   })
-
-  it("should be able to rename column for MySQL", () => {
-    const table: Table = {
```

Comment:
Is this covered as part of the table tests now?

Reply:
Okay looked closer and we do have a test for this, `"updates all the row fields for a table when a schema key is renamed"` in `table.spec.ts`